src/components/FilterDivider.tsx

```diff
@@ -0,0 +1,12 @@
+import { twMerge } from "../hooks/useComposedCssClasses";
+/**
+ * A divider component for NumericalFacets, HierarchicalFacets, and StaticFacets.
+ *
+ * @param props - A customizable className for facet divider
+ * @returns A React component for facets
+ *
+ * @public
+ */
+export function FilterDivider({ className }: { className?: string }): JSX.Element {
+  return <div className={twMerge(className, 'w-full h-px bg-gray-200 my-4')} />;
```

Comment:
should the twMerge be in the other order? I'm pretty sure we want the user defined className to have priority